luthier/config/src/tables.lua

```diff
@@ -76,15 +78,21 @@ local air_quality_dimensions = {
 
 M.physical = {
     wikiticker = {
+        dateTimeZone = "UTC",
+        type = "strict",
```

Comment:
Should document these two keys in the documentation above.

Reply:
documented. Also enriched the rest of the physical table doc